portfolio/src/main/webapp/style.css

```
@@ -7,3 +7,108 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.center {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 50%;
+}
+body {
```

[4.2.7](https://google.github.io/styleguide/htmlcssguide.html#Rule_Separation): Separate rules by new lines.

```suggestion
}

body {
```

Here and elsewhere.